submission/programs/adae.R

```diff
@@ -0,0 +1,350 @@
+# Name: ADAE
+#
+# Label: Adverse Event Analysis Dataset
+#
+# Input: ae, adsl
+# Output: adae.xpt
+#
+# Developed by : [Author/Developer name]
+
+library(admiral)
+library(admiral.test) # Contains example datasets from the CDISC pilot project
+library(curl)
+library(dplyr)
+library(tidyr)
+library(lubridate)
+library(stringr)
+library(arsenal)  
+library(diffdf)
+library(xportr)
+library(metacore)
+library(metatools)
+
+rm(list = ls())  # Code removes all the Objects in the envirnoment 
```

Comment:
Please remove. This is bad practice. Instead run the script in a fresh R session to make sure it is self-contained.

Reply:
Removed.

Reply:
Please review and resolve to ensure you do not see it on your end.

Reply:
@SHAESEN2 or @thomas-neitmann If ok, please `Resolve conversation`.